src/window/file_dialog.c

```diff
@@ -47,6 +47,8 @@ static generic_button file_buttons[] = {
 
 #define NOT_EXIST_MESSAGE_TIMEOUT 500
 
+static const int MAX_FILE_WINDOW_TEXT_WIDTH = 18 * 16;
```

Comment:
`static const int` vs. `#define`: which is better?

Reply:
I'm used to C++ and was brainwashed to believe a `static const` is always better!

I will change to `define` if you prefer though.

Reply:
I don't have a clear preference (too little in-depth experience with C), but seeing a define and a static const next to each other is a bit weird. I'm leaning towards the `static const` though, because of type safety.

Reply:
That's my reasoning, `static const` has type safety whereas `define` doesn't. But I get your point, I'll change both to `static const` to prevent OCD-induced seizures.